src/components/sbb-chip/sbb-chip.e2e.ts

```diff
@@ -1,13 +1,12 @@
-import { E2EElement, E2EPage, newE2EPage } from '@stencil/core/testing';
+import { assert, fixture } from '@open-wc/testing';
+import { html } from 'lit/static-html.js';
+import { SbbChip } from './sbb-chip';
 
 describe('sbb-chip', () => {
-  let element: E2EElement, page: E2EPage;
-
   it('renders', async () => {
-    page = await newE2EPage();
-    await page.setContent('<sbb-chip>Label</sbb-chip>');
+    await fixture(html`<sbb-chip>Label</sbb-chip>`);
```

Comment:
Just a small suggestion since `fixture` returns the root element (and you can delete row 9)

```suggestion
    const element = await fixture(html`<sbb-chip>Label</sbb-chip>`);
```